### DISCUSSION TITLE
it seems Like Hide single tabs is not working properly

### DISCUSSION THREAD

**Discussion by ochi12** [rafaelmardojai/firefox-gnome-theme#850] [General]
![image](https://github.com/user-attachments/assets/0e0948c2-1845-4730-8b2b-c1b38f373c5a)

its supposed to hide the container of that tab too and where is the add tab button now?

**Comment by rafaelmardojai** [comment]
You need to remove the Firefox view icon from the tabsbar. You can move the elements around the toolbars by right clicking the  headerbar and going to "Customize Toolbar...".
